apps/fluent-tester/src/FluentTester/TestComponents/PersonaCoin/StandardUsage.tsx

```diff
@@ -45,6 +46,19 @@ const allColors: WithUndefined<PersonaCoinFluentColor>[] = [
 
 const allPresences: WithUndefined<PersonaPresence>[] = [undefinedText, 'none', 'online', 'offline', 'busy', 'dnd', 'blocked', 'away'];
 
+const StyledPicker = (props) => {
+  const { prompt, selected, onChange, collection } = props;
+  const theme = useTheme();
+  const pickerStyles = { color: theme.colors.inputText as ColorValue, ...commonStyles.header };
+  return (
+    <Picker prompt={prompt} style={pickerStyles} selectedValue={selected} onValueChange={onChange}>
+      {collection.map((value, index) => (
+        <Picker.Item label={value} key={index} value={value} />
+      ))}
+    </Picker>
+  );
+};
+
```

Comment:
Not for this review: But I like this pattern of injecting style into an existing component. I would love to integrate this elsewhere especially because Picker is in core for win32 and not in core for all the other platforms.